pkg/p2p/server.go

```diff
@@ -832,26 +831,6 @@ func (m *MessageServer) verifyStreamMeta(streamMeta *p2p.StreamMeta) error {
 		)
 	}
 
-	if m.config.ServerVersion == "" || streamMeta.ClientVersion == "" {
```

Comment:
We should add the version check here too.

Reply:
`server` and `client` version is already guaranteed by the `owner` in the first place, we can remove it simply since this won't be hit I think.

Reply:
What about rolling upgrade? I think there are some CDC using old protocol while others using new protocol.